NBitcoin.Tests/util_tests.cs

```diff
@@ -352,6 +352,9 @@ public void util_ParseMoney()
 
 				// Attempted 63 bit overflow should fail
 				Assert.False(Money.TryParse(prefix + "92233720368.54775808", out ret));
+
+				// Attempted parsing a very small value (loosing money)  should fail
+				Assert.False(Money.TryParse(prefix + "0.00000000000001", out ret));

```

Comment:
Can you add test for `Assert.False(Money.TryParse("0.000000001"))` ? (Should fail with 9 decimal)

And `Assert.True(Money.TryParse("0.0000000000000"))` (Should work with 0)

Reply:
Done!